python/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py

```diff
@@ -83,7 +83,8 @@ def isFileTest(_file):
                 for this_file in files:
                     self.files.append(deepcopy(this_file))
             elif isType(files, IGangaFile):
-                self.files.append(deepcopy(this_file))
+                for this_file in files:
```

Comment:
I guess IGangaFiles will be in the GangaList below - so this is just for a single file?